src/bthome_ble/parser.py

```diff
@@ -275,34 +288,30 @@ def _parse_bthome_v1(
         self, service_info: BluetoothServiceInfoBleak, service_data: bytes
     ) -> bool:
         """Parser for BTHome sensors version V1"""
-        identifier = short_address(service_info.address)
-        name = service_info.name
         sw_version = 1
+        self._set_software_version(sw_version, service_info)
 
-        # Remove identifier from ATC sensors.
-        atc_identifier = (
-            service_info.address.replace("-", "").replace(":", "")[-6:].upper()
-        )
-        if name[-6:] == atc_identifier:
-            name = name[:-6].rstrip(" _")
-
-        # Try to get manufacturer
+        # Try to get manufacturer based on the name
+        identifier = short_address(service_info.address)
+        name = get_name(service_info)
         if name.startswith(("ATC", "LYWSD03MMC")):
             manufacturer = "Xiaomi"
+            # device_type = "Temperature/Humidity sensor"
```

Comment:
In v2 we use Temperature/humidity sensor. Why not here? 

And please remove the commented line

Reply:
Let’s change it, and change the tests. I have no objection.